models/products/product.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
```

Comment:
rename this to something like`product-schema` that way if you need to change the schema it easier to find the file that has it.

Reply:
k...